### ISSUE TITLE
Mesh collision not working properly

### ISSUE THREAD

**Issue by cherylwang20** [kevinzakka/obj2mjcf#31]
Hi, 

Thanks for the great package! I am a student and I was trying to use obj2mjcf to convert a beaker.obj model into sub-meshes to use as a hollow object in mujoco. 

I managed it separate it into 28 convex hulls, however, when I load the mesh into the env I have with a table and robotic arm, it penetrates the table and has its COM on the side. 

I have attached the initial setup:
![screenshot](https://github.com/kevinzakka/obj2mjcf/assets/90352960/e1a0e112-362b-42d9-80b3-ca510ff8cbec)

and after free falling from a height:
![screenshot](https://github.com/kevinzakka/obj2mjcf/assets/90352960/ae008ee5-bfd5-4308-8458-7e754694b21e)

As a reference, the box and the mug model from MuJoCo are behaving fine. 

Please see here for the xml code: 
https://github.com/cherylwang20/mj_envs/blob/3e73f238eea48cffed0423d7fcd604b467838b4a/robohive/envs/arms/ur10e/scene_gripper.xml

Any help is greatly appreciated! Thank you.

**Comment by kevinzakka** [comment]
Can you share a ZIP file containing the original mesh, the decomposition and the scene.xml with all the assets? I can take a closer look :)

**Comment by cherylwang20** [comment]
> Can you share a ZIP file containing the original mesh, the decomposition, and the scene.xml with all the assets? I can take a closer look :)

Hi Kevin, 

Thanks for the quick reply. Here is the zip file. 
[beaker_file.zip](https://github.com/kevinzakka/obj2mjcf/files/15478765/beaker_file.zip)

If you drag the scene_gripper.xml into mujoco, it should reproduce what I have. The beaker.mtl and beaker.obj are the original mesh files. the decomposition is in the folder 'beaker'. 

Thank you again and let me now if you need anything else.

**Comment by kevinzakka** [comment]
Hi @cherylwang20, the XMLs you sent were super disorganized so I stripped everything down to just the beaker and the table and tweaked the physics to get better behavior.

I found that 15 convex hulls is sufficient to approximate the beaker well. The command I ran is as follows:

```bash
obj2mjcf --obj-dir . --obj-filter beaker --save-mjcf --compile-model --decompose --overwrite --coacd-args.max-convex-hull 15
```

In terms of changes I made to the XML:

1. Switched to `elliptic` cones.
2. Changed collision `condim` to 6.
3. Stiffen contacts by reducing `solref[0]` (note it should not go below 2 x your timestep).
4. Toggled `multiccd` to get more stable contacts between the table surface and the beaker.

Cheers!

[beaker_scene.zip](https://github.com/kevinzakka/obj2mjcf/files/15494138/beaker_scene.zip)
